mikrotik/provider.go

```diff
@@ -27,6 +27,24 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("MIKROTIK_PASSWORD", nil),
 				Description: "Password for mikrotik api",
 			},
+			"tls": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("MIKROTIK_TLS", false),
+				Description: "Whether to use TLS when connecting to MikroTik or not",
+			},
+			"ca_certificate": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("MIKROTIK_CA_CERTIFICATE", ""),
+				Description: "Path to MikroTik's certificate authority",
+			},
+			"insecure": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("MIKROTIK_INSECURE", true),
```

Comment:
This should default to false. Users should have to opt out of secure TLS rather than it being the default

Reply:
Seems reasonable, done